src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -5,9 +5,10 @@ import MemoIc_Checklist from '../../../assets/icons/Ic_Checklist';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../../utils/constants';
 import {COLORS} from '../../../utils/theme';
 
-const ItemUser = ({photo, username, followed, onPress, userid}) => {
+const ItemUser = ({photo, username, followed, onPress, userid, isAnon}) => {
   const [isSelect, setIsSelect] = React.useState(false);
   const followIconFadeAnimation = React.useRef(new Animated.Value(0)).current;
+  const backgroundOpacity15PercentHexCode = '26';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `isAnon` prop is introduced to the `ItemUser` component. This prop is used to conditionally set the background color and follow icon color based on whether the user is anonymous. The change seems appropriate and doesn't introduce any issues.
```diff
- const ItemUser = ({photo, username, followed, onPress, userid}) => {
+ const ItemUser = ({photo, username, followed, onPress, userid, isAnon}) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->